cleanlab/datalab/internal/data_issues.py

```diff
@@ -23,7 +23,7 @@
 related information and statistics about the issues.
 
 The collected information can be accessed using the
-:py:meth:`get_info <cleanlab.datalab.internal.data_issues.DataIssues.get_info>` method.
+`~cleanlab.datalab.internal.data_issues.DataIssues.get_info` method.
```

Comment:
`get_info` is also present in the 'Datalab' class. Specify DataIssues.get_info using :py:meth: